mmstudio/src/main/java/org/micromanager/display/internal/displaywindow/DisplayController.java

```diff
@@ -1042,7 +1042,16 @@ public void adjustZoom(double factor) {
 
    @Override
    public void autostretch() {
-      throw new UnsupportedOperationException();
+      boolean autoStretchOn = getDisplaySettings().isAutostretchEnabled();
+      if (!autoStretchOn) {
+         // since we want to immediately switch off autostretch after applying it, we can not
+         // run this action coalesced, but need to do it directly.
+         uiController_.applyDisplaySettings(getDisplaySettings().copyBuilder().autostretch(true)
+                 .build());
+      }
+      // Strange, but the description specifies that autostretch will be switched off even when it
+      // was on before.
+      setDisplaySettings(getDisplaySettings().copyBuilder().autostretch(false).build());
```

Comment:
I agree that it is strange or at least unexpected (it also doesn't match what the GUI `Auto Once` button does).

Wouldn't it be better to change the description and make it so that this method does nothing when (continuous) autostretch is enabled? (It won't break anything because it never worked anyway :)

Reply:
Great idea!  I changed the description and made the code behave accordingly.  Will merge unless you see an issue.